src/app/shell/personal-cabinet/provider/provider-org-info/provider-org-info.component.html

```diff
@@ -1,8 +1,11 @@
-<div *ngIf="provider$ | async as provider" class="container">
-  <app-provider-info
-    [provider]="provider"
-    [currentStatus]="currentStatus"
-    (selectedTabChange)="onTabChanged($event)"
-    [isProviderView]="true"
-  ></app-provider-info>
-</div>
+<ng-container *ngIf="provider$ | async as provider">
```

Comment:
lets instead of subscription use provider, because we do have already a subscription in abstract provider component